core/src/main/java/hudson/model/BallColor.java

```diff
@@ -69,7 +69,7 @@
     ABORTED("aborted",Messages._BallColor_Aborted(), ColorPalette.DARK_GREY),
     ABORTED_ANIME("aborted_anime",Messages._BallColor_InProgress(), ColorPalette.DARK_GREY),
     NOTBUILT("nobuilt",Messages._BallColor_NotBuilt(), ColorPalette.LIGHT_GREY),
-    NOTBUILT_ANIME("nobuilt_anime",Messages._BallColor_InProgress(), ColorPalette.LIGHT_GREY),
```

Comment:
is there a reason for adding this? it's fairly common especially in JavaScript land to mandate trailing comma's, the main reason for this is that when you add a new one to the end of the list there won't be a diff on the line above.